src/rpc-call-examples/ETH/call.hurl

```diff
@@ -0,0 +1,9 @@
+POST {{ ETHEREUM RPC }}
+Content-Type: application/json
+{
```

Comment:
Also, it should be the same call as in Starknets. We can use balance of for both